obstacle_stop_planner_refine/src/node.cpp

```diff
@@ -133,7 +115,7 @@ ObstacleStopPlannerNode::ObstacleStopPlannerNode(const rclcpp::NodeOptions & nod
 
   planner_ = std::make_unique<obstacle_stop_planner::ObstacleStopPlanner>(
     this,
-    vehicle_info,
+    i,
```

Comment:
Since the scope is a bit large, I think using `vehicle_info` is better.
If you'd like to use a short name for accessing members, how about this?(adding a scope)

```cpp
  const auto vehicle_info = vehicle_info_util::VehicleInfo::create(*this);

  {
    const auto & i = vehicle_info;
    stop_param.stop_margin += i.wheel_base_m_ + i.front_overhang_m_;
    // ...
  }
  
  // ...  
  
  planner_ = std::make_unique<obstacle_stop_planner::ObstacleStopPlanner>(
    this,
    vehicle_info,
```

Reply:
It's better. I modified code 786ee28